### PR TITLE
Frontend structure

### PR DESCRIPTION
**Pull request by hroulston**
Did not finish styling for page not found. Created a new card for this task in Trello.

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import Logo from '../assets/turtle_header.jpeg'
+import '../App.css'
+class Header extends Component {
+    render() {
+        return (
+            <div className ="header">
+                <img className = "logo" src={Logo} alt="picture of a turtle" />
+                <h3> Our Header! </h3>
```
Comment (sjproctor) [inline src/components/Header.js:9]: for the most part your spacing looks really good. I would just clean up this file. Typically you don't have spaces around the equal sign of attributes.

**Review by sjproctor — APPROVED** [review]
Nice work!

File: src/components/Header.js (hunk 1 above)
Comment (hroulston) [reply src/components/Header.js:8]: Fixed!

File: src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import Logo from '../assets/turtle_header.jpeg'
+import '../App.css'
```
[the remark this answers is not in the document]
Comment (hroulston) [reply src/components/Header.js]: App.css is moved to App.js, thank you!

**Pull request merged**